terraform/groups/environments/variables.tf

```
@@ -0,0 +1,33 @@
+variable "environment" {
+  description = "The environment name to use within Concourse"
```

The 'Concourse' reference here could cause confusion since there's a possibility we may not be using CI to `plan`/`apply` changes every time (e.g. running `terraform-runner` locally during development):

```suggestion
  description = "The environment name to use"
```